pkgs/development/python-modules/apsw/default.nix

```
@@ -25,37 +24,13 @@ buildPythonPackage rec {
     sqlite
   ];
 
-  checkInputs = [
-    pytestCheckHook
-  ];
-
-  # Works around the following error by dropping the call to that function
-  #     def print_version_info(write=write):
-  # >       write("                Python " + sys.executable + " " + str(sys.version_info) + "\n")
-  # E       TypeError: 'module' object is not callable
-  preCheck = ''
-    sed -i '/print_version_info(write)/d' tests.py
+  # Project uses custom test setup to exclude some tests by default, so using pytest
+  # requires more maintenance
+  # https://github.com/rogerbinns/apsw/issues/335
+  checkPhase = ''
+    python setup.py test
   '';
```

That should be run when no checkPhase is supplied, too.

```suggestion
  checkPhase = ''
    ${python.interpreter} setup.py test
  '';
```